elbepack/cdroms.py

```diff
@@ -56,6 +57,12 @@ def mk_source_cdrom(components, codename,
     for component in components.keys():
         rfs, cache, pkg_lst = components[component]
         logging.info('Adding %s component', component)
+
+        forbiddenSrcPackages = []
+        for name, _ in cache.get_corresponding_source_packages(forbiddenPackages):
```

Comment:
Move this block outside the component loop. `forbiddenPackages` isn't changed inside the loop.

Reply:
I added it into the loop because cache change depending on the component (?) and I wasn't sure if everything is excluded properly when doing it for example only for the target component.

What would be your preferred way of handling that?